config/v1/types_ingress.go

```diff
@@ -44,12 +44,14 @@ type IngressSpec struct {
 	// +optional
 	AppsDomain string `json:"appsDomain,omitempty"`
 
-	// componentRoutes is an optional list of routes configured by the cluster-admin to
-	// override the default hostname and serving certificates. The namespace and name of
-	// each componentRoute should match an existing componentRoute defined in the status.
+	// componentRoutes is an optional list of routes that are managed by OpenShift components
+	// that a cluster-admin is able to configure the hostname and serving certificate for.
+	// The namespace and name of each componentRoute should match an existing entry in the
+	// status.ComponentRoutes list.
 	//
 	// To determine the set of configurable Routes, look at namespace and name of entries in the
-	// .status.componentRoutes, where participating operators place current route status.
+	// .status.componentRoutes list, where participating operators write the status of
+	// configurableRoutes.
```

Comment:
There is no `configurableRoutes` field.
```suggestion
	// configurable routes.
```